### PR TITLE
Rename ln as log to match Elixir Core's Conventions

### PR DESCRIPTION
**Pull request by Benjamin-Philip**
See https://github.com/elixir-nx/complex/pull/24#pullrequestreview-1274313549.

### CODE REVIEW

**Comment by Benjamin-Philip** [comment]
Forgive me if something seems idiotic in my changes.
All I did was a `sed -i 's/ln/log/g' test/*.exs lib/*.ex && mix test` for this PR.

**Comment by josevalim** [comment]
We should also keep the ln version around to avoid breaking code and deprecate it first. :)

**Comment by Benjamin-Philip** [comment]
> We should also keep the ln version around to avoid breaking code and deprecate it first. :)

Ah, I forgot again. That's why I shouldn't write sed scripts for this!

File: lib/complex.ex [hunk 1]
```diff
@@ -893,38 +893,41 @@ defmodule Complex do
 
   @doc """
   Returns a new complex that is the complex natural log of the provided
-  complex number, $ln(z) = log_e(z)$.
+  complex number, $log(z) = log_e(z)$.
```
Comment (Benjamin-Philip) [inline lib/complex.ex:896]: Should I make this

```suggestion
  complex number, $log(z) = ln(z)$.
```

?

File: lib/complex.ex [hunk 2]
```diff
@@ -1220,10 +1223,10 @@ defmodule Complex do
   def acos(z = %Complex{}) do
     i = new(0.0, 1.0)
     one = new(1.0, 0.0)
-    # result = -i*ln(z + sqrt(z*z-1.0))
-    # result = -i*ln(z + sqrt(t1))
+    # result = -i*log(z + sqrt(z*z-1.0))
+    # result = -i*log(z + sqrt(t1))
```
Comment (Benjamin-Philip) [inline lib/complex.ex:1227]: Should I leave the comments as is (i.e revert the above changes)?

**Comment by Benjamin-Philip** [comment]
@josevalim and @polvalente, Could you have a look please...

File: lib/complex.ex (hunk 1 above)
Comment (polvalente) [reply lib/complex.ex:896]: I think it should be log(z) = log_e(z)

Comment (polvalente) [reply lib/complex.ex:896]: Just so we're more explicit about what's happening

File: lib/complex.ex (hunk 2 above)
Comment (polvalente) [reply lib/complex.ex:1227]: It's fine to change them :)

**Review by polvalente — APPROVED** [review]

**Pull request merged**